rosetta-source/src/main/rosetta/product-template-type.rosetta

```diff
@@ -282,10 +281,12 @@ type PerformancePayout extends PayoutBase: <"Contains the necessary specificatio
 
 
 
-type PortfolioPriceReturnTerms extends ReturnTerms : <"Specifies an individual type of return of a Performance Payout, when such individual return is part of an aggregation of multiple similar returns, at Performance Payout net level.">
+type PortfolioReturnTerms extends ReturnTerms : <"Specifies an individual type of return of a Performance Payout, when such individual return is part of an aggregation of multiple similar returns, at Performance Payout level.">
 
+ payerReceiver PayerReceiver (1..1) <"Canonical representation of the payer and receiver parties applicable to each individual return leg.">
  underlier Product (1..1) <"Defines the product that is the subject of a tradable product definition, an underlying product definition, a physical exercise, a position, or other purposes.">
- quantity ResolvablePriceQuantity (0..1 ) <"Notional quantity of the underlier to which applies the PortfolioPriceReturnTerms.">
+ quantity NonNegativeQuantitySchedule (0..1 ) <"Notional quantity of the underlier to which applies to each individual return leg.">
```

Comment:
Remove "notional" from the description, and correct syntax typo: "to which applies" --> "which applies" (I guess)